src/input.C

```diff
@@ -1,7 +1,7 @@
 /* $Id: input.C,v 1.14 2007-12-20 22:03:56 wilsonp Exp $ */
 #include "alara.h"
 #include "input_tokens.h"
-#include "dflt_datadir.h"
+//#include "dflt_datadir.h"
```

Comment:
Have we successfully figured out how to generate this file with CMake as we were doing with autotools?

Reply:
I think we have put this info in `config.h` instead (that is current generated from `version.h.in`).  We can probably delete this line.